readme.md

```diff
@@ -98,6 +98,7 @@
 - [RubyMotion](https://github.com/motion-open-source/awesome-rubymotion) - Write cross-platform native apps for iOS, Android, macOS, tvOS, and watchOS in Ruby.
 - [Smart TV](https://github.com/vitalets/awesome-smart-tv) - Create apps for different TV platforms.
 - [GNOME](https://github.com/Kazhnuz/awesome-gnome) - Simple and distraction-free desktop environment for Linux.
+- [micro:bit](https://github.com/carlosperate/awesome-microbit) - Tiny programmable computer, designed to make learning and teaching easy and fun.
```

Comment:
Oh, and micro:bit is meant to be all lower case (I know...), which is why I haven't capitalised it.